src/lib/keyboard-shortcut/keyboard-shortcut-foundation.ts

```diff
@@ -194,4 +197,14 @@ export class KeyboardShortcutFoundation implements IKeyboardShortcutFoundation {
       }
     }
   }
+
+  /** Gets/sets the activation callback. */
+  public get activateCallback(): KeyboardShortcutActivateCallback | null | undefined {
+    return this._activateCallback;
+  }
+  public set activateCallback(value: KeyboardShortcutActivateCallback | null | undefined) {
+    if (this._activateCallback !== value) {
+      this._activateCallback = value;
+    }
```

Comment:
Just a side note, but since this doesn't have the possibility of creating an infinite loop like other properties that set host attributes, I don't typically wrap these in `if` statements.